src/elements/content-sidebar/activity-feed/task-form/TaskForm.js

```diff
@@ -61,17 +62,36 @@ type Props = TaskFormProps & TaskFormConsumerProps & InjectIntlProvidedProps;
 type TaskFormFieldName = 'taskName' | 'taskAssignees' | 'taskDueDate';
 
 type State = {|
+    addedAssignees: SelectorItems,
     approvers: SelectorItems,
     dueDate?: ?Date,
     formValidityState: { [key: TaskFormFieldName]: ?{ code: string, message: string } },
     id: string,
     isLoading: boolean,
     isValid: ?boolean,
     message: string,
+    removedAssignees: Array<TaskCollabAssignee>,
 |};
 
+function getUsersFromTask(approvers: Array<TaskCollabAssignee>): SelectorItems {
+    return (
+        approvers &&
```

Comment:
Array.isArray() is a cleaner check

Reply:
This check is actually redundant now. the block was in a higher-up component originally. now that it acts on the state of `TaskForm`, which guarantees a value exists during usage, we can remove it entirely.